admin/data/sql/migrations/20190614171658.do.restore-vewcheckdiagnostic.sql

```diff
@@ -0,0 +1,30 @@
+CREATE VIEW [mtc_admin].[vewCheckDiagnostic] AS
```

Comment:
Please make this migration idempotent.  The view now exists in some environments and not others.  This view should always be the result.

Reply:
Based on the fact that the original branch was merged prematurely, and the last commit ended up coming later... I have tried to establish the possible outcomes and hopefully addressed those now.